dom/test/browser/src/isolation.ts

```diff
@@ -626,7 +626,7 @@ describe('isolation', function() {
     function App(sources: {DOM: MainDOMSource}) {
       const triangleElement$ = sources.DOM.select('.triangle').elements();
 
-      const svgTriangle = svg({width: 150, height: 150}, [
```

Comment:
It could be that the width and height are not important for the test success/failure. I think it only affects how big the SVG looks like, and therefore it's just a visual thing.